### DISCUSSION TITLE
When a WIN PC remotely controls a MAC , two mouse cursors appear on the screen

### DISCUSSION THREAD

**Discussion by bcvb2001** [rustdesk/rustdesk#10267] [General]
### Bug Description

When I remote MAC, there are two mouse cursors on the screen again, it doesn't hide the MAC's mouse cursor, when I click on screen sharing on the status bar, it shows up normally again, it's very strange!

### How to Reproduce

Remote MAC computer

### Expected Behavior

Only one mouse cursor will appear

### Operating system(s) on local (controlling) side and remote (controlled) side

Sequoia 15.1.1

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

Version 1.3.5 (54)

### Screenshots

![40b3a1b1-1f18-4734-a08b-9b5ec5af70eb](https://github.com/user-attachments/assets/f9e13a40-3d7f-4fc0-babd-06908fd4bfcd)

**Comment by aajonaa** [comment]
I have the same problem, I feel it was my mac's system version problem, so I update the mac OS 15.1.1 to 15.2, but the problem remains and the rustdesk version of the computers are 1.3.5

**Comment by clspeter** [comment]
I have the same problem, local  `Windows11 24h2` `RustDesk 1.3.5` connect to remote `MacOS 15.2` `RustDesk 1.3.5` has this problem

**Comment by fufesou** [comment]
Reproduced.
I'll try to fix it.

**Reply by fufesou** [reply]
https://github.com/rustdesk/rustdesk/pull/10314
